test/lint.js

```diff
@@ -71,7 +79,7 @@ function load(...files) {
         }
       }
 
-      continue;
+      return;
```

Comment:
Why was this changed to `return`? 
I think we want to continue with more files, otherwise you only get the first file per folder.
See the most recent travis build: https://travis-ci.org/mdn/browser-compat-data/builds/472649710

Reply:
I accidentally screwed up a git merge.
```suggestion
      continue;
```